src/styles/Home.module.css

```diff
@@ -0,0 +1,38 @@
+.title a {
```

Comment:
go once more through all classnames - remove those that are not in use on this page pls